src/test/operators/typed_operator_base_test.hpp

```diff
@@ -0,0 +1,42 @@
+#include <boost/hana/for_each.hpp>
+
+#include "all_type_variant.hpp"
+#include "base_test.hpp"
+#include "constant_mappings.hpp"
+#include "storage/encoding_type.hpp"
+
+namespace opossum {
+
+using ParamType = std::tuple<DataType, EncodingType, bool /* nullable */>;
```

Comment:
A global alias "ParamType", even if it's just in the tests, confuses me. Can this (and `create_param_pairs` (which should really be `create_param[_tuples]`) become a member of TypedOperatorBaseTest?